edge-modules/TwinTester/Program.cs

```diff
@@ -24,7 +24,7 @@ static async Task Main()
             {
                 RegistryManager registryManager = RegistryManager.CreateFromConnectionString(Settings.Current.ServiceClientConnectionString);
 
-                using (ITwinTestInitializer twinOperator = await GetTwinOperatorAsync(registryManager, Settings.Current.AnalyzerUrl))
+                using (ITwinTestInitializer twinOperator = await GetTwinOperatorAsync(registryManager, Settings.Current.ReporterUrl))
```

Comment:
update argument name "analyzerClientUri" to "reportUrl" in GetTwinOperatorAsync method.

Reply:
should it be better to pass Settings to GetTwinOperatorAsync method?

Reply:
GetTwinOperatorAsync not uses the Settings.Current.ReporterUrl directly